docs/infrastructure/workers/worker-tools-versioning-and-caching.md

```diff
@@ -0,0 +1,73 @@
+
+The tools pulling their latest releases for Ubuntu include `wget`, `python3-pip`, `groff`, `unzip`, `apt-utils`, `curl`, `software-properties-common`, `jq`, `yq`, `openssh-client`, `rsync`, `git`, `augeas-tools`, `maven`, `gradle`, `Node 14`, `istioctl`, `linkerd `, `umoci`.
+
+For Windows these include `chocolatey` and `dotnet core 3.1.*`.
+
+If you depend on any of these packages we recommend you target the fully qualified SemVer of Worker Tools. Otherwise our additional Worker Tools tags may be suitable for your use case. We version our releases as follows:
+
+Major update
+* Update of a pinned tools major version
+* Any update of a pinned tool with a 0 Major version i.e. `0.*.*`
+* Removal of a tool
+
+Minor Update
+* Update of a pinned tools minor version
+* Addition of a new tool
+
+Patch update
+* Update of a pinned tools Patch version
+* Any new release, the latest tools will be updated automatically
+
+In short we recommend using the full `octopusdeploy/worker-tools:Major.Minor.Patch-Distro` tag format. Depending on your use case the latest releases, `octopusdeploy/worker-tools:ubuntu.18.04` and `octopusdeploy/worker-tools:windows.ltsc2019` respectivly or `octopusdeploy/worker-tools:Major-distro`, `octopusdeploy/worker-tools:Major.Minor-Distro` may be suitable for you.
+
+## Caching Worker Tools
+
+Worker Tools are cached on dynamic workers to help improve the performance of deployments. Windows workers cache the latest 2 sets of Worker Tools while Ubuntu workers cache the latest 5.
+
+To understand this cache it's important to understand a workers life-cycle. Workers are acquired from a dynamic worker pool and leased to a single cloud instance. They are allocated in a round robin fashion to individual deployment steps, storing packages, docker images and other data on disk. Workers are destroyed after either the worker has been idle for 60 minutes or has existed for 72 hours (3 days). 
+
+When a new worker is acquired if a new set of Worker Tools has been released the worker will no longer have the oldest version of Worker Tools and any other images pulled on the old worker. This is important for the performance of deployments as pull times for uncached Worker Tools are ~1.5 minutes for Ubuntu and ~20 minutes for Windows. We reccomend updating to the latest set of Worker Tools avaliable to avoid these pull times. By Caching multiple versions of Worker Tools when using the latest version any new release of Worker Tools will not result in degraded deployment performance.
```

Comment:
```suggestion
When a new worker is acquired, if a new set of Worker Tools has been released, the worker will no longer have the oldest version of Worker Tools, and any other images pulled on the old worker. This is important for the performance of deployments as pull times for uncached Worker Tools are ~1.5 minutes for Ubuntu and ~20 minutes for Windows. We recommend updating to the latest set of Worker Tools available to avoid these pull times. By Caching multiple versions of Worker Tools when using the latest version, any new release of Worker Tools will not result in degraded deployment performance.
```